wowup-electron/assets/preload/wago_test.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <h5>Test</h5>
+    <a href="https://google.com" target="_blank">Test Link</a>
```

Comment:
this supposed to be added?

Reply:
File is handy locally testing broken electron things.